connectivity/FEATURE_BLE/source/cordio/stack_adaptation/mbed_wsf_trace.c

```diff
@@ -0,0 +1,49 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2021 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if MBED_CONF_MBED_TRACE_ENABLE
+
+#include "wsf_types.h"
+#include "wsf_trace.h"
+#include "mbed-trace/mbed_trace.h"
+#include "mbed_wsf_trace.h"
+#include "platform/mbed_critical.h"
+
+#define TRACE_GROUP "BLCO"
+
+static bool_t wsf_mbed_trace_printf_cb(const uint8_t *pBuf, uint32_t len)
+{
+#if defined(MBED_TRAP_ERRORS_ENABLED) && MBED_TRAP_ERRORS_ENABLED && defined(MBED_CONF_RTOS_PRESENT)
+    if (core_util_is_isr_active() || !core_util_are_interrupts_enabled()) {
+        return FALSE;
```

Comment:
there's apparently traces in isr, I'm not routing this to an event queue as the out of order traces will be more a hindrance than help

Reply:
The only traces in ISR are issued by the baseband, serial is too slow to print them anyway.